### PR TITLE
Allow disabling of retries

### PR DESCRIPTION
**Pull request by MovieStoreGuy**
Making sure we have the configurability to disable retires.

### CODE REVIEW

**Comment by tiedotguy** [comment]
The retry pattern used throughout the code is:
```
b := backoff.NewExponentialBackOff()
b.MaxElapsedTime = maxRequestElapsedTime
for {
  if err := do(); err == nil { // always try at least once
    break
  }
  if next := b.NextBackoff(); next == backoff.Stop {
    // stop
  } else {
    // sleep
  }
}
```

It's not clear in the documentation, but a duration of 1 (nanosecond) will result in only one attempt (a duration of 0 will retry forever).

The fact that there's now a section of code with different behavior results in inconsistencies.

Please remove this and update the documentation instead.

File: README.md [hunk 1]
```diff
@@ -72,6 +72,8 @@ following configuration options:
 - `enable-http2`: boolean to enable the usage of http2 on the request.  There seems to be some incompatibility with the
   golang http2 implementation and AWS ELB/ALBs.  If you experience strange timeouts and hangs, this should be the first
   thing to disable.  Defaults to `false`
+- `enable-retires`: This will tell the forwarder to attempt to send payloads again in case it failed for the first time,
```
Comment (aelse) [inline README.md]: "retires"

File: pkg/statsd/handler_http_forwarder_v2.go [hunk 2]
```diff
@@ -110,6 +126,7 @@ func NewHttpForwarderHandlerV2(logger logrus.FieldLogger, apiEndpoint, network s
 		"client-timeout":           clientTimeout,
 		"compress":                 compress,
 		"enable-http2":             enableHttp2,
+		"enable-retries":           enableRetires,
```
Comment (aelse) [inline pkg/statsd/handler_http_forwarder_v2.go]: "retires"

File: pkg/statsd/handler_http_forwarder_v2.go [hunk 3]
```diff
@@ -162,6 +179,7 @@ func NewHttpForwarderHandlerV2(logger logrus.FieldLogger, apiEndpoint, network s
 			Transport: transport,
 			Timeout:   clientTimeout,
 		},
+		retries: enableRetires,
```
Comment (aelse) [inline pkg/statsd/handler_http_forwarder_v2.go]: "retires"

**Pull request merged**